packages/go_router/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
-## NEXT
+## 4.3.0
 
+- Adds ShellRoute for nested navigation support (flutter/flutter#99126)
+- Adds `navigatorKey` to GoRoute, which specifies the Navigator to place that
```

Comment:
Should this be `parentNavigatorKey` now?

Reply:
Good catch, fixed in fc8b9e90e34343484bcfc51a343a330a5b94931f